server/server.go

```diff
@@ -62,16 +71,85 @@ func (s *Server) Start() {
 	router := mux.NewRouter()
 
 	// Register routes with their corresponding handler funcs.
-	router.HandleFunc("/api/hello", s.helloHandler).Methods("GET")
+	router.HandleFunc("/health", s.healthHandler).Methods("GET")
+	router.HandleFunc("/ws", s.wsHandler)
 
 	// Stand up the server.
 	log.Printf("Listening on port %d....\n", s.port)
 	portAddr := fmt.Sprintf(":%d", s.port)
 	log.Fatal(http.ListenAndServe(portAddr, router))
 }
 
-// helloHandler serves requests at the /api/hello route. Responds with a
-// greeting as plain text.
-func (s *Server) helloHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintln(w, "hello world!")
+// healthHandler serves requests at the /health route. Responds with information
+// about the server's state.
+//
+// TODO: include health information about the database in response.
+func (s *Server) healthHandler(w http.ResponseWriter, r *http.Request) {
+	response := map[string]interface{}{
+		"ok":             true,
+		"numActiveGames": len(s.activeGames),
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	err := json.NewEncoder(w).Encode(response)
+	if err != nil {
+		errCode := http.StatusInternalServerError
+		errMsg := fmt.Sprintf("Failed to encode response as JSON: %v\n", err)
+		http.Error(w, errMsg, errCode)
+		return
+	}
+}
+
+// wsRequestBody represents the structure of JSON bodies in POST requests to
+// the /ws route.
+type wsRequestBody struct {
+	gameID      string
+	displayName string
+}
+
+// wsHandler serves requests at the /ws route. Creates a new game (or adds a new
+// player to the existing game that corresponds with the provided gameID), and
+// attempts to establish a Websocket connection with a client. Expects "gameID"
+// and "displayName" as query parameters when the /ws endpoint is hit.
+func (s *Server) wsHandler(w http.ResponseWriter, r *http.Request) {
+	// Extract info from query params.
+	// TODO: introduce error handling if
```

Comment:
This comment might be left over from a previous "version" of this code, too. Not sure if this belongs anymore.